docs/python/testing.md

```diff
@@ -313,6 +313,21 @@ Below are all the supported commands for testing with the Python extension in VS
 |  **Testing: Focus on Test Explorer View** | Open the Test Explorer view. Similar to **Testing: Focus on Python View** on versions prior to 2021.9.
 |  **Test: Stop Refreshing Tests** | Cancel test discovery. |
 
+## IntelliSense for pytest fixtures and parameterization
+[Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) offers IntelliSense features that can help you work more efficiently with [pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html) and [parameterized tests](https://docs.pytest.org/en/6.2.x/parametrize.html).
+
+As you're typing the parameters for your test function, Pylance will offer you a list of [auto completions](/docs/python/editing.md#autocomplete-and-intellisense) that includes known parameters from arguments defined by `@pytest.mark.parametrize`, as well as existing pytest fixtures defined in your tests file or in `confest.py`. [Code navigation](/docs/python/editing.md#navigation) features such as **Go to Definition** and **Find All References** and [rename symbol refactoring](/docs/editor/refactoring.md#rename-symbol) are also supported.
+
+![Auto completion suggestion when passing a parameter to a test function in a Python test file. The suggestion is a fixture defined in conftest.py.](/docs/python/images/testing/pytest-fixture-autocomplete.png)
```

Comment:
You can just use `images/testing/pytest-fixture-autocomplete.png` for the image paths (here and below).